### PR TITLE
Fixes issue #3754, added support for handling power status in computeresource.py

### PR DESCRIPTION
**Pull request by san7ket**
Added single power function to power on/off vm's across compute resources.
Added relevant testcases for compute resource of type RHEV.
Removed the previous implementation of power functions.

### CODE REVIEW

File: robottelo/ui/computeresource.py [hunk 1]
```diff
@@ -233,3 +206,49 @@ def vm_delete(self, res_name, vm_name, really):
...
+        there is no search button to click
+        """
+        self.click(self.search(resource_name))
+        self.click(tab_locators['resource.tab_virtual_machines'])
+        self.text_field_update(
+            locators['resource.search_filter'], vm_name)
+        strategy, value = self._search_locator()
+        return self.wait_until_element((strategy, value % vm_name))
+
+    def set_power_status(self, resource_name, vm_name, power_on=None):
```
Comment (oshtaier) [inline robottelo/ui/computeresource.py]: That is great change +1

**Comment by oshtaier** [comment]
ACK pending test results

**Comment by san7ket** [comment]
Forgot to add them, here 
root   power_unification {2} U:3 ?:14  /git_world/robottelo  1  nosetests tests.foreman.ui.test_computeresource_rhev:RhevComputeResourceTestCase.test_positive_power_off
.
----------------------------------------------------------------------
Ran 1 test in 65.259s

OK
 root   power_unification {2} U:3 ?:14  /git_world/robottelo  nosetests tests.foreman.ui.test_computeresource_rhev:RhevComputeResourceTestCase.test_positive_power_on
.
----------------------------------------------------------------------
Ran 1 test in 63.120s

OK

**Comment by oshtaier** [comment]
ACK

**Comment by kbidarkar** [comment]
ACK

**Pull request merged**